src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.PlatformNotSupported.cs

```diff
@@ -62,80 +62,80 @@ public new abstract class Arm64 : AdvSimd.Arm64
         /// </summary>
         public static unsafe Vector<long> Abs(Vector<long> value) { throw new PlatformNotSupportedException(); }
 
+        /// <summary>
+        /// svint8_t svabs[_s8]_m(svint8_t inactive, svbool_t pg, svint8_t op)
+        /// svint8_t svabs[_s8]_x(svbool_t pg, svint8_t op)
+        /// svint8_t svabs[_s8]_z(svbool_t pg, svint8_t op)
+        /// </summary>
+        public static unsafe Vector<sbyte> Abs(Vector<sbyte> value) { throw new PlatformNotSupportedException(); }
+
         /// <summary>
         /// svfloat32_t svabs[_f32]_m(svfloat32_t inactive, svbool_t pg, svfloat32_t op)
         /// svfloat32_t svabs[_f32]_x(svbool_t pg, svfloat32_t op)
         /// svfloat32_t svabs[_f32]_z(svbool_t pg, svfloat32_t op)
         /// </summary>
         public static unsafe Vector<float> Abs(Vector<float> value) { throw new PlatformNotSupportedException(); }
```

Comment:
Is this order using .NET names for types (i.e. Single instead of float)? I'm guessing that's why `Vector<float> Abs` is at the end?

Reply:
This is using the order specified [here](https://github.com/dotnet/runtime/pull/104704#discussion_r1674831126).
So, yes the names instead of the types.